ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md

```diff
@@ -68,13 +68,18 @@ Für die Ressource Slot MUSS die REST-Interaktion "READ" implementiert werden.
 
     Anwendungshinweise: Weitere Informationen zur Suche nach "_id" finden sich in der [FHIR-Basisspezifikation - Abschnitt "Parameters for all resources"]https://hl7.org/fhir/R4/search.html#all).
 
-2. Der Suchparameter "schedule" MUSS unterstützt werden:
+2. Der Suchparameter "schedule" MUSS in Kombination mit einem definierten Startzeitpunkt mittels Suchparameter "start"   unterstützt werden. Der Suchparameter "schedule" KANN alleinstehend unterstützt werden.
+Beim Fehlen des start-Suchparameters SOLL der aktuelle Zeitpunkt des Servers als Startzeitpunkt per Default verwendet werden (Slots aus der Vergangenheit wären somit nur dann zu liefern, wenn ein Client Slot.start explizit auf ein Datum aus der Vergangenheit setzt).
```

Comment:
```suggestion
Beim Fehlen des "start"-Suchparameters SOLL der aktuelle Zeitpunkt des Servers als Startzeitpunkt per Default verwendet werden. Slots aus der Vergangenheit sind entsprechend zurückzugeben, falls ein Client Slot.start explizit auf ein Datum aus der Vergangenheit setzt. Die Rückgabe von Slots aus der Vergangenheit mit einem anderen Status außer "busy" ist optional.
```

Reply:
@alexzautke aus dem zweiten "SOLL" würde ich sogar ein "KANN" machen, da ich es für die Terminbuchung (UC) unwesentlich halte Termine aus der Vergangenheit (egal ob busy oder frei) exponieren zu können. So etwas wie "Ein User wertet aus, ob in der Vergangenheit noch Slots frei gewesen wäre (z.B. für analytics Zwecke)", sehe ich ich nicht im primären Scope des Moduls Terminplanung.
Hast Du eine Meinung @alexey-tschudnowsky?

Reply:
question: für Stufe 3 und 4 wäre hier ggf. ein "MUSS" für Anforderung zum "aktuelle Zeitpunkt des Servers als Startzeitpunkt" angebracht

Reply:
@alexzautke zur info: wir haben uns dafür entschieden nur den ersten Satz zu übernehmen für den Suchparameter, da sich die zwei anderen auf andere Suchparameter beziehen und das nicht das ursprüngliche Problem der Ticket-Anfrage adressieren.